tests/run-make/core-ffi-typecheck-clang/rmake.rs

```diff
@@ -0,0 +1,194 @@
+//@ needs-force-clang-based-tests
+// This test checks that the clang defines for each target allign with the core ffi types defined in
+// mod.rs. Therefore each rust target is queried and the clang defines for each target are read and
+// compared to the core sizes to verify all types and sizes allign at buildtime.
+//
+// If this test fails because Rust adds a target that Clang does not support, this target should be
+// added to SKIPPED_TARGETS.
+
+use run_make_support::{clang, regex, rfs, rustc, serde_json};
+use serde_json::Value;
+
+// It is not possible to run the Rust test-suite on these targets.
```

Comment:
Suggestion: could we elaborate *why* we can't run this test against these targets?